tests/unit_tests/test_source_mesh.py

```diff
@@ -97,49 +97,48 @@ def test_unstructured_mesh_sampling(model, request, test_cases):
     source = openmc.IndependentSource(space=space, energy=energy)
     model.settings.source = source
 
-    with cdtemp([mesh_filename]):
-        model.export_to_xml()
+    model.export_to_xml()
```

Comment:
Why was the `with` statement removed above? Without it, running the tests will pollute the directory with a bunch of input/output files that don't get cleaned up.

Reply:
Committed that by accident -- I was putting together a mini-app for mapping an `IndependentSource` sites onto a mesh and needed the outputs as sample problem. Reverting now!